src/JarbField.tsx

```diff
@@ -135,31 +132,35 @@ export class JarbField extends React.Component<Props, State> {
         if (field === 'number' && fieldConstraints.fractionLength && fieldConstraints.fractionLength > 0) {
           if (this.patternValidator === null) {
             const regex = patterns.fractionNumberRegex(fieldConstraints.fractionLength);
-            this.patternValidator = validators.pattern(label, regex)
+            this.patternValidator = validators.pattern(label, regex);
           }
 
           enhancedValidate.push(this.patternValidator);
         } else if (field === 'number') {
           if (this.patternValidator === null) {
-            this.patternValidator = validators.pattern(label, patterns.numberRegex)
+            this.patternValidator = validators.pattern(label, patterns.numberRegex);
           }
 
           enhancedValidate.push(this.patternValidator);
         }
       } else {
-        console.warn(`jarb-redux-form: constaints for "${validator}" not found, but a JarbField was rendered, this should not occur, check your validator. See: https://github.com/42BV/jarb-redux-form/issues/4`);
+        console.warn(
+          `jarb-redux-form: constraints for "${validator}" not found, but a JarbField was rendered, this should not occur, check your validator. See: https://github.com/42BV/jarb-redux-form/issues/4`,
+        );
       }
     } else {
-      console.warn('jarb-redux-form: constraints are empty, but a JarbField was rendered, this should not occur, make sure the constraints are loaded before the form is displayed. See: https://github.com/42BV/jarb-redux-form/issues/3');
+      console.warn(
+        'jarb-redux-form: constraints are empty, but a JarbField was rendered, this should not occur, make sure the constraints are loaded before the form is displayed. See: https://github.com/42BV/jarb-redux-form/issues/3',
+      );
     }
 
     return enhancedValidate;
   }
 
-  render() {
+  public render(): JSX.Element {
```

Comment:
Vanwaar de typing?

Reply:
Door de regel eslint-typescript/[explicit-function-return-type](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md). Misschien een beetje overkill in dit scenario maar naar mijn mening over het algemeen wel een handige regel om te hebben. 

Ziet er overigens naar uit dat het volgens de standaard [types van react](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c06a6662f8d1e4dfbaf12c3cf1da7175b636824a/types/react/index.d.ts#L422) `ReactNode` moet zijn.